event_routing_backends/processors/xapi/event_transformers/navigation_events.py

```diff
@@ -58,7 +58,7 @@ def get_object(self):
             `Activity`
         """
         return Activity(
-            id=self.get_data('data.target_url'),
+            id=self.get_data('data.target_url', True),
```

Comment:
I made this required since it's a required attribute and will fail downstream without it.

Reply:
I was debugging this one hahah. Have in mind that if the error is too big (like for this tasks with 10000 arguments) mysql cannot save all that information:
```}
_mysql.connection.query(self, query)
django.db.utils.OperationalError: (1153, "Got a packet bigger than 'max_allowed_packet' bytes")
```